src/App.tsx

```diff
@@ -1,29 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { ItemList } from './components/ItemsList';
```

Comment:
It seems like there is a typo in the import statement. The component is likely exported as `ItemsList` but is being imported as `ItemList`. Please correct the import to match the exported component name.